tesla_http_proxy/DOCS.md

```diff
@@ -54,10 +119,23 @@ nxjvrm2M6uKfDEYS52ITVVbzqGMzzbKCO/tuu78432jU6Z96BNR8NSoRXg==
 -----END PUBLIC KEY-----
 ```
 
-You should have a config file at `/share/nginx_proxy/nginx_tesla.conf` that hosts the static public key file.  You may need to modify this file depending on your SSL config.  
+- You should have a config file at `/share/nginx_proxy/nginx_tesla.conf` that hosts the static public key file. You may need to modify this file depending on your SSL config.
```

Comment:
The config file doesn't host the static pub key file but has a setting to specify the path to the pub/priv key. Unless I'm wrong this probably need to be rephrased.

Reply:
Fixed, it's a author's suggestion